src/app/webhook/components/webhook.component.html

```diff
@@ -0,0 +1,50 @@
+<plex-layout [main]='(encontro || nuevoHook) ? "8": "12"'>
+    <plex-layout-main>
+        <plex-title titulo="Webhooks">
+            <plex-button label="Nuevo" type="primary" (click)="nuevo()" ></plex-button>
+        </plex-title>
+    <div class="d-flex justify-content-between align-items-end">
```

Comment:
```suggestion
    <div justify>
```

Referencia: http://ui.andes.gob.ar/components/justify/api

Aclaración: Quedo un error en la página de UI.

Reply:
Ya corrí el error en la documentación.